easybuild/easyconfigs/c/Cube/Cube-4.2-gompi-1.4.12-no-OFED.eb

```diff
@@ -20,19 +20,21 @@ description = """Cube, which is used as performance report explorer for Scalasca
 
 toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
 
-# http://apps.fz-juelich.de/scalasca/releases/cube/4.2/dist/cube-4.2.tar.gz
 sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/4.2/dist']
+source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
 
 patches = ['Cube-4.2_fix-Qt-version-check.patch']
 
 dependencies = [('Qt', '4.8.4')]
 
-configopts = "QMAKE=$EBROOTQT/bin/qmake MOC=$EBROOTQT/bin/moc UIC=$EBROOTQT/bin/uic"
-
 sanity_check_paths = {
     'files': ["bin/cube", "lib/libcube4.a", "lib/libcube4.so"],
     'dirs': ["include/cube", "include/cubew"],
 }
 
+# If all prerequisites for the Cube Java reader are available on the system,
+# parallel builds will fail. Alternatively, disable the Java reader using
+# configopts = "--without-java-reader"
```

Comment:
wouldn't it be better to just disable this then, since you told me only Tau uses this, and it is shipping it's own `.jar` for this?

either that, or include Java as a (build?) dependency here